.fastly/config.toml

```diff
@@ -1,12 +1,13 @@
-config_version = 2
+config_version = 3
 
 [fastly]
 api_endpoint = "https://api.fastly.com"
 
 [language]
 [language.go]
-tinygo_constraint = ">= 0.26.0-0" # NOTE -0 indicates to the CLI's semver package that we accept pre-releases (TinyGo users commonly use pre-releases).
-toolchain_constraint = ">= 1.18"
+tinygo_constraint = ">= 0.26.0-0"       # NOTE -0 indicates to the CLI's semver package that we accept pre-releases (TinyGo users commonly use pre-releases).
```

Comment:
I mentioned this in https://github.com/fastly/compute-starter-kit-go-default/pull/8 but version 0.2.0 of the SDK brings the TinyGo requirement up to >= 0.28.1.